web/src/components/CheckboxGroup.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Box, Checkbox, Label } from 'rebass';
+
+const CheckboxGroup = ({ input, meta, options }) => {
+  const { name, onChange, value: inputVal } = input;
+  const { touched, error } = meta;
+
+  const checkboxes = options.map(({ label, value }, idx) => {
+    const handleChange = e => {
+      const { checked } = e.target;
+      const vals = [...inputVal];
+
+      if (checked) vals.push(value);
+      else vals.splice(vals.indexOf(value), 1);
+
+      return onChange(vals);
+    };
+
+    return (
+      <Label key={`checkbox-${value}`}>
+        <Checkbox
+          name={`${name}[${idx}]`}
+          value={value}
+          checked={inputVal.includes(value)}
```

Comment:
Does `Array.includes` get polyfilled or transpiled by Babel?  I know it's not supported by IE, which we probably have to support, so wanted to flag it.

Reply:
good call, need to add `babel-polyfill` (https://babeljs.io/docs/usage/polyfill/)